images/amd64.dockerfile

```diff
@@ -12,24 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.20@sha256:52921e63cc544c79c111db1d8461d8ab9070992d9c636e1573176642690c14b5 as cnibuilder
+FROM golang:1.20 as cnibuilder
 COPY . /usr/src/afxdp_k8s_plugins
 WORKDIR /usr/src/afxdp_k8s_plugins
-RUN apt-get update && apt-get -y install --no-install-recommends libbpf-dev=1:0.3-2 \
-       && apt-get -y install --no-install-recommends clang=1:11.0-51+nmu5 llvm=1:11.0-51+nmu5 gcc-multilib=4:10.2.1-1 \
-       && make buildcni
+RUN apt-get update
+RUN apt-get update && apt-get -y install --no-install-recommends libbpf-dev=1:1.1.0-1
+RUN apt-get update && apt-get -y install --no-install-recommends libxdp-dev=1.3.1-1
+RUN apt-get -y install -o APT::Keep-Downloaded-Packages=false --no-install-recommends clang=1:14.0-55.6  
+RUN apt-get -y install -o APT::Keep-Downloaded-Packages=false --no-install-recommends llvm=1:14.0-55.6
+RUN apt-get -y install -o APT::Keep-Downloaded-Packages=false --no-install-recommends gcc-multilib=4:12.2.0-3 
+RUN make buildcni
 
-FROM golang:1.20-alpine@sha256:87d0a3309b34e2ca732efd69fb899d3c420d3382370fd6e7e6d2cb5c930f27f9 as dpbuilder
+FROM golang:1.20-alpine as dpbuilder
 COPY . /usr/src/afxdp_k8s_plugins
 WORKDIR /usr/src/afxdp_k8s_plugins
-RUN apk add --no-cache build-base~=0.5 libbsd-dev~=0.11 \
-      && apk add --no-cache libbpf-dev~=0.5 --repository=https://dl-cdn.alpinelinux.org/alpine/v3.15/community \
-      && apk add --no-cache llvm~=15.0.7-r0 clang~=15.0.7-r0 \
-	  && make builddp
+RUN apk add --no-cache build-base~=0.5-r3 
+RUN apk add --no-cache libbsd-dev~=0.11.7 
+# TODO: Add version later 
+RUN apk add --no-cache libxdp-dev 
+RUN apk add --no-cache libbpf-dev

```

Comment:
remove comment, add versions

again if moving to libxdp, remove libbpf

combine all these RUNs into a singe RUN